src/persist/src/unreliable.rs

```diff
@@ -210,6 +217,157 @@ where
     }
 }
 
+/// An unreliable delegate to [BlobMulti].
+#[derive(Debug)]
+pub struct UnreliableBlobMulti<B> {
+    rng: tokio::sync::Mutex<SmallRng>,
+    blob: B,
+}
+
+impl<B: BlobMulti + Send + Sync> UnreliableBlobMulti<B> {
```

Comment:
I think this one and the one below might not be strictly necessary? Or did you check with some example? Only the `BlobMulti` impl below and the `Consensus` impl need the constraint.

Reply:
are you talking about the `<B: BlobMulti + Send + Sync>` bounds?

if so, I go back and forth on this. should each block have the minimum set of things it needs to compile? or (like this is) should it have the minimum set of things for it to be useful? e.g. is it better to allow you to construct this but then you get a "method not found error" if you try to call one of the methods that _do_ need the bounds? I don't really have a strong opinion either way, so happy to defer to your preference

Reply:
Ah yes sorry, that part of the message stayed in my head. 🙈 

And yes, I also don't know what constraints to put here. It's a bit like `HashMap`, which you can construct with something that is not `Hash`, but then later it tells you that you can't insert things because inserting has the bound. Seems like people put the minimal constraints that are needed, at least when looking at `HashMap`, but I'd leave it as is after all.

Reply:
oh Hashmap is a good precedent!

> I'd leave it as is after all

ha! I'd already pushed the change when you said this. does "as-is" mean the original way or the new HashMap way :D?

Reply:
As is was referring to the original state. But my own example seems convincing to me 😅

Reply:
works for me! :D